README.md

```diff
@@ -38,47 +38,52 @@ Also, check [Danger's guide on Phrasing](http://danger.systems/js/usage/culture.
 
 ## What is currently being checked
 
-### PR 
+### PR
 
 - [x] Fail when no description is provided
 - [x] Warns if description provided is too short
 - [x] Big PRs
 
 ### Platform/Language agnostic
 
-- [ ] Warn if some files/folders to be changed/committed like `.gitignore`, `Gemfile`, `Gemfile.lock`, `.travis.yml`
-- [ ] `>>>` Strings to make sure rebase was successful
+- [x] `>>>` Strings to make sure rebase was successful
 - [ ] Warn when Amazon Secret Key is hardcoded
-- [ ] Warn when `Dangerfile` was modified
+- [x] Warn when `Dangerfile` was modified
 - [x] Warn when `http://` is used
 - [x] Warn when definition files were changes but their lockfile weren't:
   - `Gemfile` vs `Gemfile.lock`
   - `package.json` vs `yarn.lock`
+- [x] Warn if some files/folders were changed/committed:
+  - `Dangerfile`
+  - `.gitignore`
+  - `Gemfile`
+  - `Gemfile.lock`
+  - `.travis.yml`
+  - `coverage`
 
 ### Node
 
-- [x] Warn if some files/folders to be changed/committed:
-  - `yarn.lock`
-  - `npm-shrinkwrap.json`
+- [x] Warn when `npm install -g` is used
+- [x] Warn when `console.log` is added
+- [ ] Warn when `package.json` was modified and `yarn.lock` or `shrinkwrap` was not
+- [x] Warn if node version is different between .travis.yml, .nvmrc, package.json and README (or just warn if node version has change just in one of these locations)
+- [ ] At packages.json every package should have its version fixed (do not use ^ or ~), or explicitly set the major and minor versions (ie.: 1.2.x)
+- [ ] [TypeScript] Warn if using `<any>` as return type.
+- [x] Warn if some files/folders were changed/committed:
+  - `yarn.lock`,
```

Comment:
Is there any particular reason why yarn.lock files should trigger a warn if they are committed?

Reply:
No reason... I will remove it since it is enough to warn when `package.json` changes but `yarn.lock` doesn't